packages/react-components/react-text/src/stories/Text/TextTypography.stories.tsx

```diff
@@ -1,5 +1,16 @@
 import * as React from 'react';
-import { Body1, Caption1, Display, Subtitle1, LargeTitle, Title1, Title2, Title3, Subtitle2, Caption2 } from '../index';
+import {
+  Body1,
+  Caption1,
+  Display,
+  Subtitle1,
+  LargeTitle,
+  Title1,
+  Title2,
+  Title3,
+  Subtitle2,
+  Caption2,
```

Comment:
_nit:_ You could go the extra mile and order these alphabetically

```suggestion
  Body1,
  Caption1,
  Caption2,
  Display,
  LargeTitle,
  Subtitle1,
  Subtitle2,
  Title1,
  Title2,
  Title3,
```